serviceinsight/upgrades/support-policy.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Support Policy
```

Comment:
We have too many docs with the very same title, makes it hard to find the one you want!
```suggestion
title: ServiceInsight support policy
```